ntoskrnl/ex/handle.c

```diff
@@ -68,12 +68,20 @@ ExpLookupHandleTableEntry(IN PHANDLE_TABLE HandleTable,
             /* Get the mid level pointer array */
             PointerArray = PointerArray[Handle.HighIndex];
 
+            /* TODO: maybe ASSERT? */
+			if (PointerArray == NULL)
```

Comment:
These seem to be symptoms of Mm bugs, so we should figure out why some pages seem to be mapped improperly.